### ISSUE TITLE
Example FCNXS -Data Shuffling does not happen

### ISSUE THREAD

**Issue by Viswa14** [dmlc/mxnet#1683]
I am working with FCN-XS model and I find average train accuracy and validation accuracy to be constant for all epochs. On experimenting by printing img_name and val_name in data.py, i find for each epochs all the batches get same set of data, but I do not find this reasonable. Shouldn't we shuffle images  for each epoch ? Can you high light where the shuffling happens ? I cannot find shuffle parameter in fit () defined in solver.py. Can you please help. Thank you

@tornadomeet  @zhaw @tqchen

**Comment by tornadomeet** [comment]
yes, we'd better shuffle the data every epoch! because here we input the file list to data.py, so we should shuffle the list file ```flist ```after each epoch, these needs to change some code of data.py(data.py just here is very simple)
